.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -1,224 +1,283 @@
 ---
-format_version: '8'
-default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: android
 trigger_map:
-  - push_branch: develop
-    workflow: develop
 
-  - push_branch: release/*
-    workflow: release
+- push_branch: develop
+  workflow: develop
 
-  - push_branch: master
-    workflow: master
+- push_branch: release/*
+  workflow: release
 
-  - push_branch: '*'
-    workflow: primary
+- push_branch: master
+  workflow: master
+  
+- push_branch: "*"
+  workflow: automatic_code_review
```

Comment:
which kind of editor config are you using to edit and format this file? I'm using the default VSCode and it seems the old format looks good. I tried to re-format the whole file and everything back to the previous format. Could you please re-format again then we could focus on the main content of Danger config?

![image](https://user-images.githubusercontent.com/16315358/135996780-b6c0a80f-01f1-46fc-b6f9-d98c12742209.png)

Reply:
@luongvo nice question, I used the Bitrise's website editor view and this is how they are indented now, I suggest we keep it to Bitrise style since whenever we try to make a change to the Bitrise's workflow using the website GUI, the indentation will be automatically updated to my current indentation style 🙏

Reply:
@minhnimble nope, I don't think so, we need to manage it as repository source code and everything should be formatted in the same way.